src/components/playback/playbackmanager.js

```diff
@@ -2319,8 +2319,11 @@ class PlaybackManager {
                     return;
                 }
 
-                rankStreamType(prevSource.DefaultAudioStreamIndex, prevSource, mediaSource, 'Audio');
-                rankStreamType(prevSource.DefaultSubtitleStreamIndex, prevSource, mediaSource, 'Subtitle');
+                if (audio)
+                    rankStreamType(prevSource.DefaultAudioStreamIndex, prevSource, mediaSource, 'Audio');
+
+                if (subtitle)
+                    rankStreamType(prevSource.DefaultSubtitleStreamIndex, prevSource, mediaSource, 'Subtitle');
```

Comment:
afaik, most of the code is Google styled.
iirc, we allow simple body at the same line.
```suggestion
                if (audio) {
                    rankStreamType(prevSource.DefaultAudioStreamIndex, prevSource, mediaSource, 'Audio');
                }

                if (subtitle) {
                    rankStreamType(prevSource.DefaultSubtitleStreamIndex, prevSource, mediaSource, 'Subtitle');
                }
```